unstract/connectors/src/unstract/connectors/filesystems/minio/static/json_schema.json

```
@@ -35,7 +36,8 @@
     "region_name": {
       "type": "string",
       "title": "Region Name",
-      "description": "Region name of AWS S3 account."
+      "default": "default-region",
```

```suggestion
      "default": "",
```
Go for either an empty string / a commonly used AWS default such as `us-east`